### PR TITLE
Update linux travis jobs with xspec 12.10.1

### PR DESCRIPTION
**Pull request by olaurino**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,11 +16,11 @@ matrix:
       sudo: required
       dist: trusty
     # As above, Python 3.6, setup.py install, xspec 12.10
-    - env: XSPECVER="12.10.0e" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2 TRAVIS_PYTHON_VERSION="3.6"
+    - env: XSPECVER="12.10.1b" NUMPYVER="1.11" FITS="astropy" INSTALL_TYPE=install TEST=submodule MATPLOTLIBVER=2 TRAVIS_PYTHON_VERSION="3.6"
       sudo: required
       dist: trusty
     # As above, python 3.7, numpy 1.14, matplotlib 2.0
```
Comment (DougBurke) [inline .travis.yml]: Can we have 'matplotlib 2.0' changed to 'matplotlib 3' here ;-) Assuming that it works with mpl v3.

Comment (olaurino) [reply .travis.yml]: just did it in #590

Comment (DougBurke) [reply .travis.yml]: but I'm looking at #590 and I don't see it in the repository

Comment (DougBurke) [reply .travis.yml]: e.g. https://github.com/sherpa/sherpa/pull/590/commits/478f0f49f7b49f6995e120737ddb05fe31ced857

note I'm talking about the comment not the actual version used in the run..

Comment (olaurino) [reply .travis.yml]: sorry, I meant #591

Comment (DougBurke) [reply .travis.yml]: Aha - I think you meant #591

**Comment by olaurino** [comment]
It's kind of stupid to have two PRs for this (especially considering I made a mistake and the jobs are not succeeding right away). Closing in favor of #591.

**Pull request closed without merging**